migration/migration_pkg_test.go

```diff
@@ -100,3 +104,51 @@ func TestRecalculateBloomFilters(t *testing.T) {
 		assert.Equal(t, core.EventsBloom(b.Receipts), b.EventsBloom)
 	}
 }
+
+func TestChangeTrieNodeEncoding(t *testing.T) {
+	testdb := pebble.NewMemTest()
+	t.Cleanup(func() {
+		require.NoError(t, testdb.Close())
+	})
+
+	buckets := []db.Bucket{db.ClassesTrie, db.StateTrie, db.ContractStorage}
+
+	var n trieNode
+	require.NoError(t, testdb.Update(func(txn db.Transaction) error {
+		for _, bucket := range buckets {
```

Comment:
Can we set the root key for the bucket here? Should improve coverage a tad and be more realistic.

Reply:
Done